EFCore.CheckConstraints/Internal/EnumCheckConstraintConvention.cs

```diff
@@ -123,15 +123,15 @@ private bool TryParseContiguousRange(ValueConverter? converter, IEnumerable valu
         return success;
     }
 
-    private static bool TryGetMinMax<T>(IEnumerable values, out T minValue, out T maxValue)
+    private static bool TryGetMinMax<T>(object[] values, out T minValue, out T maxValue)
         where T : INumber<T>
     {
-        var enumValues = values.Cast<T>().Distinct().ToList();
+        var enumValues = values.Cast<T>().ToArray();
```

Comment:
I've always been more of a fan of `.ToList()`, but that's just preference.

Reply:
I tend to prefer ToArray when I know the length is fixed and I won't need to add/remove elements... But yeah, it doesn't really matter.

Any other comments on this PR?

Reply:
Well this is weird, maybe I'm not used to Github's interface, but I had a few other comments but I can only click "Add Review Comment" instead of a normal comment. Perhaps that means something different than what I thought. I'll make actual them actual comments so you can see them.

Reply:
Oh I think I figured it out!